src/main/java/org/myrobotlab/service/Runtime.java

```diff
@@ -1878,6 +1874,14 @@ public Runtime(String n) {
     log.info("java.vm.name [{}]", System.getProperty("java.vm.name"));
     log.info("java.vm.vendor [{}]", System.getProperty("java.vm.vendor"));
     log.info("java.specification.version [{}]", System.getProperty("java.specification.version"));
+    
+    
+    String vmVersion = System.getProperty("java.specification.version");
+    vmVersion = "11";

```

Comment:
This is just a warning , right?   I use java11 with MRL ...  java8 is pretty much dead.   So I guess, I don't like this message here... we should be able to run under java8 or java11..  (and we do..)  consider removing this.

Reply:
well .. all the build, and all my test machines are currently running java8 
But ya .. java 11 is the future -  I've had at least 3 people send me noworkys (one being Nick_S) and when I told them to uninstall java11 and install java 8 - they are worky ... 
So until we have more worky java11 it should stay .. and for longer term - we should have a tested jvm version, and if mrl see's that its not on that version it should minimally warn